docs/src/data/ignoredLinks.ts

```diff
@@ -11,4 +11,9 @@ export const IGNORED_LINKS = [
   'https://docs.aws.amazon.com/amplify/latest/userguide/getting-started.html', // 403
   'https://docs.aws.amazon.com/cognito/latest/developerguide/user-pool-settings-attributes.html', // 403
   'https://github.com/aws-amplify/amplify-ui/issues/new/choose', // 302. Only at test. Can't reproduce the status code. Can't find the redirected link.
+  'https://github.com/aws-amplify/amplify-ui-android/issues/new/choose', // 302
+  'https://github.com/aws-amplify/amplify-ui-swift/issues/new/choose', // 302
+  'https://github.com/aws-amplify/amplify-ui-swift-liveness/issues/new/choose', // 302
+  'https://developer.apple.com/design/human-interface-guidelines/patterns/accessing-private-data/', // 401 Apple doesn't like bots maybe
+  'https://youtu.be/JfzwVl1wy0s', // 303, Fix after Liveness release to use expanded url
```

Comment:
Adding this here temporarily. We didn't want to delay the docs publish for Liveness. Also added the swift and android issue urls.

Reply:
Instead of ignoring the link, we should update it to https://www.youtube.com/watch?v=JfzwVl1wy0s&feature=youtu.be

Reply:
Updated to add the missing rekognition links; we'll address the YT url prior to this PR and after liveness docs launch.